var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -197,6 +197,13 @@ class Sundials(CMakePackage):
     depends_on('superlu-mt+blas', when='+superlu-mt')
 
     # ==========================================================================
+    # Patches
+    # ==========================================================================
+
+    # remove OpenMP header file and function from hypre vector test code
+    patch('test_nvector_parhyp.patch', when='@2.7.0:')
```

Comment:
if we patch is already merged upstream, you can limit it to `@2.7.0:3.0.0` so that it won't apply to `3.0.1` or alike when it's out.

Reply:
@gardner48: is the patch already merged upstream?  or should the range be open like this?

Reply:
This is patched in the next release. I've added an upper limit on the version range.